Client/Frontend/Browser/PageZoom/PageZoomView.swift

```diff
@@ -0,0 +1,181 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveUI
+import WebKit
+import Data
+import Shared
+import BraveShared
+
+private struct ZoomView: View {
+  @ScaledMetric private var buttonWidth = 44.0
+  
+  var minValue: Double
+  var maxValue: Double
+  @Binding var value: Double
+  
+  var onDecrement: () -> Void
+  var onReset: () -> Void
+  var onIncrement: () -> Void
+  
+  var body: some View {
+    HStack(spacing: 0.0) {
+      Button(action: onDecrement) {
+        Image(systemName: "minus")
+          .font(.system(.footnote).weight(.medium))
+          .foregroundColor(Color(UIColor.braveLabel))
+          .frame(width: buttonWidth)
+      }
+      .disabled(value == minValue)
+      
+      Divider()
+      
+      Text(NSNumber(value: 0.999), formatter: PageZoomView.percentFormatter)
+        .font(.system(.footnote).weight(.medium))
+        .fixedSize(horizontal: true, vertical: false)
+        .hidden()
+        .overlay(resetZoomButton)
+        .padding(.horizontal)
+      
+      Divider()
+      
+      Button(action: onIncrement) {
+        Image(systemName: "plus")
+          .font(.system(.footnote).weight(.medium))
+          .foregroundColor(Color(UIColor.braveLabel))
+          .frame(width: buttonWidth)
+      }
+      .disabled(value == maxValue)
+    }
+    .padding(.vertical, 8.0)
+    .fixedSize(horizontal: false, vertical: true)
+    .background(Color(UIColor.secondaryBraveBackground))
+    .clipShape(RoundedRectangle(cornerRadius: 10.0, style: .continuous))
+  }
+  
+  private var resetZoomButton: some View {
+    Button(action: onReset) {
+      Text(NSNumber(value: value), formatter: PageZoomView.percentFormatter)
+        .font(.system(.footnote).weight(.medium))
+        .foregroundColor(Color(UIColor.braveLabel))
+        .padding()
+        .contentShape(Rectangle())
+    }
+    .fixedSize(horizontal: true, vertical: false)
+    .disabled(value == 1.0)
```

Comment:
Does reset == 1 always? Or does reset set to default in settings? If its default this should be checking that not 1

Reply:
Changed it. It used to be just 1.